### DISCUSSION TITLE
Caused stuck with Remmina when automatically switch theme (with noficiation?)

### DISCUSSION THREAD

**Discussion by no1xsyzy** [AutoDarkMode/Windows-Auto-Night-Mode#673] [General]
It just happened for the second time today, so I haven't tried to reproduce it. However, those two times happens after the same operation, and the operation only happens twice, meaning it 100% causes this issue. So, I guess it can reproduce.
More exploration is probably required.

### Computers
Computer A: Windows 10, with AutoNightMode. Notify before automatically switching themes is ON
Computer E: Linux, with Remmina + RDP plugin.

### Possible steps for reproduce
1. Locked Computer A for the night
2. Unlock Computer A in the morning (checked email), the theme switch notification showed up. Let it delay 15 minutes as I had configured
3. Lock Computer A again
4. Wait at least that delay
5. Remote control Computer A from Computer E
6. It keeps showing "Please wait" for at least one hour (I guess it would be forever) and I cannot control Computer A.

### Misc
I am not sure whether other RDP clients can cause the same bug.
Since the switch notification shows only twice a day, I think it take long time to check any reproduction.
I am not sure about where the bug comes from. But I think preventing to switch theme when locked could be a workaround.

**Comment by Spiritreader** [comment]
I'm not really sure what we're supposed to do to fix it sadly.
Sounds more like something is up with the implementation of that RDP protocol.

Again, the theme switching procedure is entirely controlled by Windows and what we're doing is using theme switching out of specification. 
Thus some first and, more likely, third party tools might glitch out.

Also please note that you can disable theme switches while the lock screen is active on Windows 10 by unchecking this box via settings:
![image](https://user-images.githubusercontent.com/18438899/212893738-b68409e3-91e4-4ed3-88fa-ab2d4d1801aa.png)

**Reply by no1xsyzy** [reply]
I didn't notice that, thank you.

I started the issue for future record, and see if anyone have any idea about reproducing and/or preventing it. Sorry I didn't notice GitHub has discussion function, it fits here more.
It causes inconvenience for me when this happens, so I can not intentionally trigger this. And fortunately it happens rarely. You can think it is just waiting more information in very long term.
If it is not good to put future records here at random time, tell me so I would find another place for my own records.
